kubectl-rook

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
From [this discussion](https://github.com/rook/kubectl-rook-ceph/issues/2#issuecomment-921111474) I thought we decided on calling the tool `kubectl-rook-ceph`?